packages/zoe/test/unitTests/zcf/test-zcf.js

```diff
@@ -584,3 +583,268 @@ test(`zcf.makeZCFMint - burnLosses - seat exited`, async t => {
+  const userSeat = await E(zoe).offer(invitation, proposal, payments);
+  return { zcfSeat, userSeat };
+};
+
+const sameAsNormalZCFSeat = async (t, emptySeat, normalSeat) => {
+  // Note: not exhaustive
+  t.deepEqual(Object.keys(emptySeat), Object.keys(normalSeat));
+  t.deepEqual(emptySeat.getProposal(), normalSeat.getProposal());
+  t.deepEqual(
+    emptySeat.getCurrentAllocation(),
+    normalSeat.getCurrentAllocation(),
+  );
+  t.is(emptySeat.hasExited(), normalSeat.hasExited());
+};
+
+const sameAsNormalUserSeat = async (t, emptySeat, normalSeat) => {
+  // Note: not exhaustive
+  t.deepEqual(Object.keys(emptySeat), Object.keys(normalSeat));
+  t.deepEqual(await emptySeat.getProposal(), await normalSeat.getProposal());
+  t.deepEqual(
+    await emptySeat.getCurrentAllocation(),
+    await normalSeat.getCurrentAllocation(),
+  );
+  t.is(await emptySeat.hasExited(), await normalSeat.hasExited());
+  t.is(await emptySeat.getOfferResult(), await normalSeat.getOfferResult());
+};
+
+test(`zcf.makeEmptySeatKit`, async t => {
+  const { zcf, zoe } = await setupZCFTest();
+  const result = zcf.makeEmptySeatKit();
+  t.deepEqual(Object.keys(result), ['zcfSeat', 'userSeat']);
+  const { zcfSeat: zcfSeatActual, userSeat: userSeatActualP } = result;
+  const {
+    zcfSeat: zcfSeatExpected,
+    userSeat: userSeatExpected,
+  } = await makeOffer(zoe, zcf);
+  await sameAsNormalZCFSeat(t, zcfSeatActual, zcfSeatExpected);
+  const userSeatActual = await userSeatActualP;
+  await sameAsNormalUserSeat(t, userSeatActual, userSeatExpected);
+  // Currently seats made with the "makeEmptySeatKit" method cannot
+  // have any give or want because we cannot yet escrow or reallocate
+  // assets as part of the offer.
+  // TODO: add `give` and `want` to seats made on the contract/ZCF side
+  // https://github.com/Agoric/agoric-sdk/issues/1724
+});
+
+test(`zcfSeat from zcf.makeEmptySeatKit - only these properties exist`, async t => {
+  const expectedMethods = [
+    'exit',
+    'kickOut',
+    'getNotifier',
+    'hasExited',
+    'getProposal',
+    'getAmountAllocated',
+    'getCurrentAllocation',
+    'isOfferSafe',
+    'stage',
+  ];
+  const { zcf } = await setupZCFTest();
+  const makeZCFSeat = () => zcf.makeEmptySeatKit().zcfSeat;
+  const seat = makeZCFSeat();
+  t.deepEqual(Object.keys(seat), expectedMethods);
+});
+
+test(`zcfSeat.getProposal from zcf.makeEmptySeatKit`, async t => {
+  const { zcf } = await setupZCFTest();
+  const makeZCFSeat = () => zcf.makeEmptySeatKit().zcfSeat;
+  const seat = makeZCFSeat();
+  t.deepEqual(seat.getProposal(), {
+    exit: {
+      onDemand: null,
+    },
+    give: {},
+    want: {},
+  });
+});
+
+test(`zcfSeat.hasExited, exit from zcf.makeEmptySeatKit`, async t => {
+  const { zcf } = await setupZCFTest();
+  const { zcfSeat, userSeat } = zcf.makeEmptySeatKit();
+  t.falsy(zcfSeat.hasExited());
+  zcfSeat.exit();
+  t.truthy(zcfSeat.hasExited());
+  t.deepEqual(await E(userSeat).getPayouts(), {});
```

Comment:
check `userSeat.hasExited()` as well?

Reply:
Ok, I will make an issue for this bug that will be solved by your PR, and I will marking the tests as `test.failing`